src/test/java/nl/quintor/studybits/indy/wrapper/MessageScenarioIT.java

```diff
@@ -320,28 +336,32 @@ public void fullScenarioTest() throws Exception {
     }
 
     public static void onboardIssuer(TrustAnchor steward, Issuer newcomer) throws InterruptedException, ExecutionException, IndyException, IOException {
+        // Create Codecs to facilitate encryption/decryption
+        MessageEnvelopeCodec stewardCodec = new MessageEnvelopeCodec(steward);
+        MessageEnvelopeCodec newcomerCodec = new MessageEnvelopeCodec(newcomer);
+
         // Connecting newcomer with Steward
 
         // #Step 4.1.2 & 4.1.3
         // Create new DID (For steward_faber connection) and send NYM request to ledger
-        String governmentConnectionRequest = steward.createConnectionRequest(newcomer.getName(), "TRUST_ANCHOR")
-                .thenApply(connectionRequest -> new MessageEnvelope<>(IndyMessageTypes.CONNECTION_REQUEST, connectionRequest, null, steward, null)).get().toJSON();
+        String governmentConnectionRequest = new MessageEnvelopeCodec(null).encryptMessage(steward.createConnectionRequest(newcomer.getName(), "TRUST_ANCHOR").get(),
```

Comment:
Just a comment: Its weird to use encryptMessage as method name when creating a plain text message